ingestion/src/metadata/ingestion/source/storage/s3/metadata.py

```diff
@@ -334,25 +334,6 @@ def _generate_unstructured_container(
             sourceUrl=self._get_bucket_source_url(bucket_name=bucket_response.name),
         )
 
-    @staticmethod
-    def _manifest_entries_to_metadata_entries_by_bucket(
```

Comment:
moving to the StorageService itself and keeping the global manifest load in the `init`